### PR TITLE
feat(dw): renaming instances of workload to dw

### PR DESCRIPTION
**Pull request by gitdallas**
Closes: #[RHOAIENG-4748](https://issues.redhat.com//browse/RHOAIENG-4748)

## Description
renamed several instances of "workload" to "distributed workload"
image

## How Has This Been Tested?
tests updated where necessary, visually ran through UI

## Test Impact
tests update to match ui changes

## Request review criteria:

Self checklist (all need to be checked):
- [ x] The developer has manually tested the changes and verified that the changes work
- [ x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ x] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx [hunk 1]
```diff
@@ -76,13 +76,13 @@ export const DWWorkloadsTable: React.FC = () => {
...
     <Card>
-      <CardTitle>Workloads</CardTitle>
+      <CardTitle>Distributed Workload</CardTitle>
       <CardBody>
         <Table
           enablePagination
           data={workloads.data}
           columns={columns}
-          emptyTableView={<>No workloads match your filters</>}
+          emptyTableView={<>No distributed workload match your filters</>}
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:85]: workloads should be plural here

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx [hunk 2]
```diff
@@ -76,13 +76,13 @@ export const DWWorkloadsTable: React.FC = () => {
 
   return (
     <Card>
-      <CardTitle>Workloads</CardTitle>
+      <CardTitle>Distributed Workload</CardTitle>
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:79]: Workloads should be plural here

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const DWWorkloadsTable: React.FC = () => {
     return (
       <Card isFullHeight>
         <EmptyStateErrorMessage
-          title="Error loading workloads"
+          title="Error loading distributed workload"
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:25]: workloads should be plural here

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx [hunk 4]
```diff
@@ -68,7 +68,7 @@ export const DWStatusOverviewDonutChart: React.FC = () => {
               right: 140, // Adjusted to accommodate legend
               top: 20,
             }}
-            subTitle="Workloads"
+            subTitle="Distributed Workload"
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx:71]: Workloads should be plural here

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx [hunk 5]
```diff
@@ -18,7 +18,7 @@ export const DWStatusOverviewDonutChart: React.FC = () => {
     return (
       <Card isFullHeight>
         <EmptyStateErrorMessage
-          title="Error loading workloads"
+          title="Error loading distributed workload"
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWStatusOverviewDonutChart.tsx:21]: workloads should be plural here

File: frontend/src/__tests__/cypress/cypress/e2e/distributedWorkloads/GlobalDistributedWorkloads.cy.ts [hunk 6]
```diff
@@ -172,15 +172,15 @@ describe('Workload Metrics', () => {
...
   });
 
   it('Should render the workloads table with empty state', () => {
     initIntercepts({ hasWorkloads: false });
     globalDistributedWorkloads.visit();
 
-    cy.findByLabelText('Workload status tab').click();
-    cy.findByText('No workloads match your filters').should('exist');
+    cy.findByLabelText('Distributed workload status tab').click();
+    cy.findByText('No distributed workload match your filters').should('exist');
```
Comment (mturley) [inline frontend/src/__tests__/cypress/cypress/e2e/distributedWorkloads/GlobalDistributedWorkloads.cy.ts:184]: plural here

**Review by mturley — COMMENTED** [review]
A few instances where I think we want to retain the plural "workloads" here. Also we should probably put screenshots in the PR that cover all the places the text changed and just make sure we run it by @xianli123 and @cfullam1

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx (hunk 1 above)
Comment (xianli123) [reply frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:85]: Hi @mturley I am not sure about this. I will bring this to Breda and get back to you asap.

Comment (xianli123) [reply frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:85]: @mturley I asked Katie, the content designer, for this. She suggested that we use "Distributed workload metrics" for now. She thought we could use the single form to stay consistent with other areas (specifically "Project metrics"), but if we get any pushback we can revisit it. Does it make sense?

**Comment by xianli123** [comment]
Hi @mturley I remembered I left a comment for the naming. But I can't find it. I would repeat the answer that was from Katie, the content designer. She suggested that we could use "Distributed workload metrics" for now. We could use the single form to stay consistent with other areas (specifically "Project metrics"), but if we get any pushback we can revisit it. What do you think?

**Comment by gitdallas** [comment]
@xianli123 - your comment is [here](https://github.com/opendatahub-io/odh-dashboard/pull/2631#discussion_r1540410369)

and the change is reflected [here](https://github.com/opendatahub-io/odh-dashboard/pull/2631/files#diff-9da8a4c6deab2566b922319c468a2f2db6537135bff3db48aac22b158ba932bcR85)

**Comment by xianli123** [comment]
@gitdallas Thanks. It looks good.

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx [hunk 7]
```diff
@@ -76,13 +76,13 @@ export const DWWorkloadsTable: React.FC = () => {
...
     <Card>
-      <CardTitle>Workloads</CardTitle>
+      <CardTitle>Distributed Workloads</CardTitle>
       <CardBody>
         <Table
           enablePagination
           data={workloads.data}
           columns={columns}
-          emptyTableView={<>No workloads match your filters</>}
+          emptyTableView={<>No distributed workload metrics match your filters</>}
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:85]: Well, in this specific case this is a table of Distributed Workloads, not metrics, so I think it should be "No distributed workloads match your filters". @xianli123 does that make sense in the context of the empty state here inside the Distributed Workloads table? "Distributed Workload Metrics" makes sense for the page title and top-level descriptions/errors, but here in this table the workloads themselves are the objects being listed/filtered.

File: frontend/src/__tests__/cypress/cypress/e2e/distributedWorkloads/GlobalDistributedWorkloads.cy.ts [hunk 8]
```diff
@@ -191,16 +191,16 @@ describe('Workload Metrics', () => {
...
   });
 
   it('Should render the workloads table with empty state', () => {
     initIntercepts({ workloads: [] });
     globalDistributedWorkloads.visit();
 
-    cy.findByLabelText('Workload status tab').click();
-    cy.findByText('No workloads match your filters').should('exist');
+    cy.findByLabelText('Distributed workload status tab').click();
+    cy.findByText('No distributed workload metrics match your filters').should('exist');
```
Comment (mturley) [inline frontend/src/__tests__/cypress/cypress/e2e/distributedWorkloads/GlobalDistributedWorkloads.cy.ts:203]: (We'd have to change it here too)

**Review by mturley — COMMENTED** [review]
Sorry for the delay in re-reviewing this @gitdallas. I have one point of pushback that I'm curious about @xianli123's opinion on.

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx (hunk 7 above)
Comment (xianli123) [reply frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:85]: Is this for the "no results" state? If so, I agree with @mturley 's opinion. "No distributed workloads match your filters" is better. cc @gitdallas

Comment (mturley) [reply frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:85]: That's correct

**Review by mturley — APPROVED** [review]

**Comment by mturley** [comment]
FYI this will have some conflicts with https://github.com/opendatahub-io/odh-dashboard/pull/2650 because I moved the cypress tests around there, sorry about that. should be simple to resolve

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**